tests/probes/yamlfilecontent/test_probes_yamlfilecontent_array.xml

```diff
@@ -80,7 +80,7 @@
     </ind-def:yamlfilecontent_state>
 
     <ind-def:yamlfilecontent_state version="1" id="oval:0:ste:2">
-      <ind-def:value_of datatype="string" var_ref="oval:0:var:2"/>
+      <ind-def:value_of datatype="string" var_ref="oval:0:var:2" var_check="at least one" />
```

Comment:
So the returning item is a `yamlfilecontent_state` with two entities `value_of`, each with one value?
I thought the returning item would be a `yamlfilecontent_state`, with one entity `value_of` and two values.

We should check behavior of `xmlfilecontent`.

Reply:
Great finding, from the specs the value_of element in xmlsfilecontent_item element is 0 to unbounded, and in the probe there is a for cycle that adds value_of elements, so you seem to be right. I will look into that, it will simplify the code.

Reply:
@yuumasato Hopefully fixed in 42337d4b35ceb006421cd44c8564b81ed6c75c99

Reply:
> Hopefully fixed in [42337d4](https://github.com/OpenSCAP/openscap/commit/42337d4b35ceb006421cd44c8564b81ed6c75c99)

Makes sense to me.

I got a bit confused now, for clarity and record, before commit mentioned above, how was the returned item?
2 `item`s with 1 entity `value_of` and 1 `value`?

Reply:
Before  42337d4 it was 2 `yamlfilecontent_item` elements, each with 1 `value_of` child element, but now it's 1 `yamlfilecontent_item` element with 2 `value_of` child elements.

Reply:
Thanks for the clarification.